### PR TITLE
Check manual examples for all chapters even if one fails

### LINKED ISSUE #1138: New test of manual examples fails too fast
The test added by @markuspf in #1082 fails a little bit too fast, as can be seen e.g. at https://travis-ci.org/gap-system/gap/jobs/200748202. It would be useful to run it through all chapters to see where the diffs are, rather then stop after the first failing chapter.

### PR DESCRIPTION
**Pull request by alex-konovalov**
Closes #1138

### CODE REVIEW

**Comment by alex-konovalov** [comment]
At the moment, PR #1135 is good to check this after it will be merged.

**Comment by alex-konovalov** [comment]
This PR has been tested with some artificial failures (two broken examples in two chapters) and in this case it worked as intended. It should be ready to merge as soon as the current CI tests will finish.

**Comment by alex-konovalov** [comment]
Puzzling where the -0.02% reduction of coverage comes from...

File: etc/ci.sh [hunk 1]
```diff
@@ -69,14 +76,17 @@ testmanuals)
...
 GAPInput
 
+    TESTMANUALSPASS=yes
     for ch in tst/testmanuals/*.tst
     do
-        bin/gap.sh -q -L testmanuals.wsp --cover $COVDIR/$(basename $ch).coverage <<GAPInput
-        TestManualChapter("$ch");
-        QUIT_GAP(0);
-GAPInput
+        TestManualChapter $ch
```
Comment (fingolfin) [inline etc/ci.sh]: A simpler way to implement this (which possibly avoids the need of creating `TestManualChapter`), using the `||` operator:
```
bin/gap.sh -q -L testmanuals.wsp --cover $COVDIR/$(basename $ch).coverage <<GAPInput || TESTMANUALSPASS=no
...
```
Or of course also
```
echo "TestManualChapter(\"$1\"); QUIT_GAP(0);" | bin/gap.sh ... || TESTMANUALSPASS=no
```

**Comment by alex-konovalov** [comment]
@fingolfin Thanks for the tip - I've updated the PR and run CI tests twice - with and without errors. This is ready for merge now.

**Pull request merged**